main/mainworker.cpp

```diff
@@ -12307,12 +12331,13 @@ bool MainWorker::SwitchLightInt(const std::vector<std::string>& sd, std::string
 		if (!IsTesting) {
 			//send to internal for now (later we use the ACK)
 			lcmd.RADIATOR1.subtype = sTypeSmartwaresSwitchRadiator;
-			PushAndWaitRxMessage(m_hardwaredevices[hindex], (const uint8_t *)&lcmd, nullptr, -1, User.c_str());
+			PushAndWaitRxMessage(m_hardwaredevices[hindex], (const uint8_t*)&lcmd, nullptr, -1, User.c_str());
 		}
 		return true;
+	}
 	case pTypeGeneralSwitch:
 	{
-
+		tRBUF lcmd;
```

Comment:
I don't think this variable declaration is needed?

Reply:
It was since I've added braces in case above